consensus/src/block_graph.rs

```diff
@@ -1829,7 +1838,7 @@ impl BlockGraph {
         block_id: BlockId,
         block: Block,
         operation_set: OperationHashMap<(usize, u64)>,
-        endorsement_ids: Vec<EndorsementId>,
+        endorsement_ids: EndorsementHashMap<u32>,
```

Comment:
As well as the parameters are increasing. We should wrap block_id, operation_set, endorsement_ids, and (may be) block in a structure. IncomingBlock?

Reply:
Incoming block is a consensus struct and theses structs come from protocol. Maybe that's the beginning of a refactoring on how to manage blocks?